packages/babel-plugin-proposal-pipeline-operator/test/fixtures/minimal/optimize-0-param-arrow/output.js

```diff
@@ -1,3 +1,5 @@
+var _ref, _a;
```

Comment:
It seems that these aren't needed :thinking:

Reply:
Nope... I'm almost sure that they're an artifact of declaring the placeholder in the visitor and then throwing it away during optimization.

Reply:
Found a quick solution in 29fa385. Let me know what you think.